### PR TITLE
Add VAT french format

### PR DESCRIPTION
**Pull request by baptistedonaux**

### CODE REVIEW

File: src/Faker/Provider/fr_FR/Payment.php [hunk 1]
```diff
@@ -5,6 +5,28 @@
...
+     *
+     * @return string VAT Number
+     */
+    public function vat($spacedNationalPrefix = true)
+    {
+        $prefix = ($spacedNationalPrefix) ? "FR " : "FR";
+
+        $company = new Company($this->generator);
+
+        return sprintf("%s%s%s%s", $prefix, self::randomNumber(2, true), $company->siren($spacedNationalPrefix));
```
Comment (fzaninotto) [inline src/Faker/Provider/fr_FR/Payment.php]: no, don't instantiate `Company` by hand, instead, use `this->siren` directly.

**Comment by fzaninotto** [comment]
Could you add a mention of the new formatter in the locale-specific section of the README file?

**Comment by baptistedonaux** [comment]
Good for you or missing details ? https://github.com/fzaninotto/Faker/pull/1241/commits/904d977d324108a6b346902a9738457dab18272e

**Comment by fzaninotto** [comment]
Thanks!

**Comment by baptistedonaux** [comment]
ping @fzaninotto

**Comment by t11n** [comment]
Does not seem to work for me:
`PHP Fatal error:  Call to undefined method Faker\Provider\fr_FR\Payment::siren() in /var/www/udboffice-app/vendor/fzaninotto/faker/src/Faker/Provider/fr_FR/Payment.php on line 24`

**Comment by fzaninotto** [comment]
Because the change wasn't already released

**Pull request merged**